content/akeneo-event-platform/best-practices.md

```diff
@@ -0,0 +1,29 @@
+# Best practices
+
+## Queuing Events
+
+To maintain performance with a high volume of subscriptions, the event platform requires a response within the specified time limit, please refer to [this page](/akeneo-event-platform/concepts.html#ack-timeout) to know more. Handle received events using a queue or asynchronously to ensure you acknowledge the event quickly, before processing it.
+
+## Handling 429 Responses
+
+The platform can send a large number of events in a short time, potentially causing overload and leading to `500` responses from your side. In such cases, your subscription may be suspended. By implementing a proper `429` response mechanism, you can delay event processing to maintain service stability and avoid suspension.
+
+<!-- TODO rework paragraph after this PR is merged -->
+<!-- https://github.com/akeneo/event-platform/pull/144 -->
+
+## Suspending and Resuming Subscriptions During Migration
+
+If you plan to perform a data migration, suspend your subscription to avoid to avoid receiving events [you cannot handle](/akeneo-event-platform/concepts.html#retry-policy). Once the migration is complete, simply resume your subscription to receive new events. Events triggered when your subscription was suspended will not be sent. This ensures a smooth transition and uninterrupted service.
```

Comment:
Still not comfortable with the idea here :) 

For me it's more: "if you're able to plan a unavailability of your subscribing service, then you can decide to suspend/resume to avoid seeing your subscription to be revoked, in any case, during the time the subscription is suspended, you will loose new events".

Reply:
If you are unable to guarantee a positive response when you receive events (e.g. during infrastructure maintenance) you may decide to suspend/resume your subscription to avoid it being suspended in accordance with our policy.
While your subscription is suspended, no events will be sent to you and will not be transmitted once your subscription is resumed.